CONTRIBUTING.md

```diff
@@ -80,17 +80,18 @@ Please contribute! The Layer5 site uses Gatsby. The process of contributing to d
 1. Get a local clone of your fork of the site.
 `git clone https://github.com/layer5io/layer5`
 1. Switch to the `master` branch.
-`git checkout master`
+`git switch master`
```

Comment:
Why this change?

Reply:
@Nikhil-Ladha actually if we don't do `git switch master` the previous commits of a different branch also get included into the new one.

Reply:
Which branch and what commits? Example?

Reply:
For example, if I'm already working on this repo and I create a branch to work on some issue, I do the changes and make a PR, meanwhile I have to work on another issue and so I create another branch to work on, before creating a new branch if I don't do `git switch master` all my changes from the previous branch would also come onto the new branch.

Reply:
Though ig we would probably not need this information here considering this is for getting started, but then we could also get rid of the `git checkout master` here cause after cloning we would be on the master branch itself 🤔, right? @Nikhil-Ladha

Reply:
That's obvious.
But, the same can be done with `git checkout master`

Reply:
Oh, I didn't know that😅, I've actually always done `git switch master` just to be safe.

Reply:
Ok, so the thing is `git checkout master` does exactly everything of `git switch master` in fact, git` switch` is a (code)copy of `git checkout master`. `git switch` was added considering the dual nature of `git checkout` to create branches and restore changes. And, therefore to avoid confusions 2 new commands `git switch` and `git restore` was added.

Reply:
> Oh, I didn't know thatsweat_smile, I've actually always done git switch master just to be safe.
It's fine. Just, that the such PRs should have a better commit-msg and description to justify the changes being done.

Reply:
Aah okay, makes sense, yes you're right.